app/models/ride.rb

```diff
@@ -1,3 +1,5 @@
 class Ride < ApplicationRecord
   belongs_to :amusement_park
+  has_many :mechanic_rides
+  has_many :mechanics, through: :mechanic_rides
 end
```

Comment:
Great job with these associations. It looks like you are missing the relationship tests for these associations, however.